common/sgx/tcbinfo.c

```diff
@@ -595,6 +595,14 @@ static oe_result_t _read_tcb_info(
         OE_TRACE_VERBOSE("V2: Reading tcbType");
         OE_CHECK(_read_property_name_and_colon("tcbType", itr, end));
         OE_CHECK(_read_integer(itr, end, &value));
+
+        // HW representation of CPUSVN for a given FMSPc is not architecturally
+        // defined to provide designers more flexibility. SW needs "tcbType" to
```

Comment:
Is "tcpType" part of the FMSPC?

Reply:
No, tcbType is a field added in v2 collateral
https://api.portal.trustedservices.intel.com/documentation#pcs-certificate